app/models/miq_server.rb

```diff
@@ -328,6 +332,7 @@ def monitor
     Benchmark.realtime_block(:log_active_servers)      { log_active_servers }               if threshold_exceeded?(:server_log_frequency, now)
     Benchmark.realtime_block(:worker_monitor)          { monitor_workers }                  if threshold_exceeded?(:worker_monitor_frequency, now)
     Benchmark.realtime_block(:worker_dequeue)          { populate_queue_messages }          if threshold_exceeded?(:worker_dequeue_frequency, now)
+    monitor_myself
```

Comment:
Because this is a comparison of a column to a threshold value, I thought benchmarks and a threshold check, to avoid running this too often, was overkill.

Reply:
This will already happen on a `5.seconds` timer anyway, so I think that is fine.